app/src/main/res/values/styles.xml

```diff
@@ -1,8 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+
```

Comment:
Remove this extra blank line

Reply:
please refer the document for the spacing
https://docs.google.com/document/d/1AHIJagMfATgpbNCFmYoYaSqzsKp8JjTgPLQFXEFJdUA/edit?usp=sharing